requirements.txt

```diff
@@ -13,4 +13,8 @@ pytest==7.4.2
 Werkzeug==3.0.0
 Flask-WTF==1.2.1
 Bootstrap-Flask==2.3.2
-email_validator==2.0.0
\ No newline at end of file
+email_validator==2.0.0
+python-docx==1.0.1
+typing-extensions==4.8.0
+lxml==4.9.3
+matplotlib==3.5.3
```

Comment:
Why are you using `matplotlib==3.5.3` and `python-docx==1.0.1` when there are newer versions?